specification/maps/data-plane/Render/stable/2024-04-01/render.json

```diff
@@ -0,0 +1,1022 @@
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/map/statetile": {
+      "get": {
+        "summary": "Use to get state tiles in vector format that can then be used to display feature state information in an indoor map.",
+        "description": "\n\nFetches state tiles in vector format typically to be integrated into indoor maps module of map control or SDK. The map control will call this API after user turns on dynamic styling (see [Zoom Levels and Tile Grid](/azure/location-based-services/zoom-levels-and-tile-grid))",
+        "operationId": "Render_GetMapStateTile",
+        "x-ms-examples": {
+          "Successful State Tile Request": {
+            "$ref": "./examples/Render_GetMapStateTile.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/zTileIndex"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/xTileIndex"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/yTileIndex"
+          },
+          {
+            "name": "statesetId",
+            "in": "query",
+            "description": "The stateset id.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": [
+          "application/vnd.mapbox-vector-tile",
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "This tile is returned from a successful Get Map State Tile call",
+            "schema": {
+              "type": "object",
+              "format": "file",
+              "readOnly": true
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "description": "The content-type for the response."
+              }
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/map/copyright/caption/{format}": {
+      "get": {
+        "summary": "Use to get copyright information to use when rendering a tile.",
+        "description": "\n\nThe `Get Copyright Caption` API is an HTTP GET request designed to serve copyright information to be used in conjunction with tiles requested from the Render service. In addition to a basic copyright for the whole map, it can serve specific groups of copyrights for some countries/regions.\n\nAs an alternative to copyrights for map request, one can receive captions for displaying the map provider information on the map.",
```

Comment:
```suggestion
        "description": "\n\nThe `Get Copyright Caption` API is an HTTP GET request designed to serve copyright information to be used with tiles requested from the Render service. In addition to a basic copyright for the whole map, it can serve specific groups of copyrights for some countries/regions.\n\nAs an alternative to copyrights for map request, it can also return captions for displaying provider information on the map.",
```